SQL/database_changelog.txt

```diff
@@ -1,13 +1,21 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 5.17; The query to update the schema revision table is:
+The latest database version is 5.19; The query to update the schema revision table is:
```

Comment:
wow bro wow

Reply:
someone noticed forget who i think watermelon

Reply:
there is another pr that edits the database version to 5.19, im gonna count this as 5.20